themes/nord.json

```diff
@@ -238,7 +236,7 @@
       "name": "Comment",
       "scope": "comment",
       "settings": {
-        "foreground": "#4C566A"
+        "foreground": "#aaddff40"
```

Comment:
I think this is a personal change so please it, there is also already a ongoing change to improve the comment color contrast/brightness at arcticicestudio/nord#94.